core/src/main/java/hudson/util/RunList.java

```diff
@@ -230,12 +221,8 @@ public boolean apply(@Nullable T r) {
      * @since 1.544
      * @deprecated use {@link #filter(Predicate)}
```

Comment:
remove as well then?

Reply:
@oleg-nenashev I wish **REALLY** :)  but this comment https://github.com/jenkinsci/jenkins/pull/5344#discussion_r589122029